spec/features/bulk_updates/bulk_updates_science_keywords_spec.rb

```diff
@@ -293,11 +295,16 @@
         end
 
         it 'displays the new keyword' do
-          within '.science-keywords-preview' do
+          within '#metadata-preview' do
             expect(page).to have_no_content('EARTH SCIENCE ATMOSPHERIC TEMPERATURE SURFACE TEMPERATURE', normalize_ws: true)
-            expect(page).to have_content('EARTH SCIENCE SOLID EARTH ROCKS/MINERALS/CRYSTALS SEDIMENTARY ROCKS', normalize_ws: true)
-
-            expect(page).to have_content('EARTH SCIENCE ATMOSPHERE AEROSOLS', normalize_ws: true)
+            expect(page).to have_content('EARTH SCIENCE', normalize_ws: true)
```

Comment:
this should say, have_no_content for SURFACE_TEMPERATURE and the rest should say has_content for the other keywords.